<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResNorm</class>
 <widget class="QWidget" name="ResNorm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>880</width>
        <height>580</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="verticalSpacing">
          <number>6</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="lblResolution">
           <property name="text">
            <string>Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="MantidQt::MantidWidgets::DataSelector" name="dsResolution" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoLoad" stdset="0">
            <bool>true</bool>
           </property>
           <property name="loadLabelText" stdset="0">
            <string>Load</string>
           </property>
           <property name="workspaceSuffixes" stdset="0">
            <stringlist>
             <string>_res</string>
            </stringlist>
           </property>
           <property name="fileBrowserSuffixes" stdset="0">
            <stringlist>
             <string>_res.nxs</string>
            </stringlist>
           </property>
           <property name="showLoad" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lblVanadium">
           <property name="text">
            <string>Vanadium:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="MantidQt::MantidWidgets::DataSelector" name="dsVanadium" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="autoLoad" stdset="0">
            <bool>true</bool>
           </property>
           <property name="workspaceSuffixes" stdset="0">
            <stringlist>
             <string>_red</string>
             <string>_sqw</string>
            </stringlist>
           </property>
           <property name="fileBrowserSuffixes" stdset="0">
            <stringlist>
             <string>_red.nxs</string>
             <string>_sqw.nxs</string>
            </stringlist>
           </property>
           <property name="showLoad" stdset="0">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="treeSpace"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="plotPane">
           <item>
            <widget class="MantidQt::MantidWidgets::PreviewPlot" name="ppPlot" native="true">
             <property name="showLegend" stdset="0">
              <bool>true</bool>
             </property>
             <property name="canvasColour" stdset="0">
              <color>
               <red>255</red>
               <green>255</green>
               <blue>255</blue>
              </color>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="underPlotSpace">
             <item>
              <widget class="QLabel" name="lblPreviewSpec">
               <property name="text">
                <string>Preview Spectrum: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spPreviewSpectrum">
               <property name="maximum">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="legendSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbPlotCurrent">
               <property name="text">
                <string>Plot Current Preview</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="gbRun">
            <property name="title">
             <string>Run</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>7</number>
             </property>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>385</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="pbRun">
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>384</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_44">
            <item>
             <widget class="QGroupBox" name="gbOutput">
              <property name="title">
               <string>Output Options</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="lblPlot">
                 <property name="text">
                  <string>Plot Result: </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="cbPlot">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>75</width>
                   <height>0</height>
                  </size>
                 </property>
                 <item>
                  <property name="text">
                   <string>All</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Intensity</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Stretch</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbPlot">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Plot</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_18">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="pbSave">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Save Result</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Common/DataSelector.h</header>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::PreviewPlot</class>
   <extends>QWidget</extends>
   <header>MantidQtWidgets/Plotting/PreviewPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
